### PR TITLE
fix: tweaked the duration and font of the wooexpress setup messages

### PR DESCRIPTION
**Pull request by rjchow**
Related to #73872

## Proposed Changes

* Adjusted the timings and messages and subtitle font for the WooExpress setup stepper

## Testing Instructions

- Set up the repo and load http://calypso.localhost:3000/setup/wooexpress
- Or use the live link
- Observe that the loading titles should roughly correspond to:
    1. Woo! We're creating your store
    2. Building the foundations
    3. Organizing the stock room
    4. Applying the finishing touches
    5. Turning on the lights (may or may not see this depending on how long it takes to create the site)
    6. Opening the doors (may or may not see this depending on how long it takes to create the site)

Should look like this:

https://user-images.githubusercontent.com/27843274/222679870-a20b05f3-b3df-4045-b5a3-1b922b15af8c.mov

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-processing-loading-messages.ts [hunk 1]
```diff
@@ -23,17 +23,36 @@ export function useProcessingLoadingMessages( flow?: string | null ): LoadingMes
...
 
 	if ( isWooExpressFlow( flow || null ) ) {
-		return stepData.currentStep === 'siteCreationStep'
-			? [
+		switch ( stepData.currentStep ) {
+			case 'siteCreationStep':
+				return [
 					{
-						title: __( 'Woo! We’re creating your store' ),
+						title: __( "Woo! We're creating your store" ),
```
Comment (chihsuan) [inline client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-processing-loading-messages.ts:31]: Thanks for the fix @rjchow! 

I think we also need to change the following texts:

https://github.com/Automattic/wp-calypso/blob/46129e3b81e8cc0a7af78757aa4a99dfd143dd28/client/landing/stepper/declarative-flow/internals/steps-repository/assign-trial-plan/index.tsx#L52-L54

https://github.com/Automattic/wp-calypso/blob/46129e3b81e8cc0a7af78757aa4a99dfd143dd28/client/landing/stepper/declarative-flow/internals/steps-repository/site-creation-step/index.tsx#L153

Comment (rjchow) [reply client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-processing-loading-messages.ts:31]: 👀 👀 

I didn't actually realise that I changed the apostrophe! Nor did I expect that the same string is repeated in a few places

Thanks! Will amend this

File: client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-processing-loading-messages.tsx [hunk 2]
```diff
@@ -23,31 +24,91 @@ export function useProcessingLoadingMessages( flow?: string | null ): LoadingMes
...
+				return [
 					{
 						title: __( 'Applying the finishing touches' ),
-						subtitle: __(
-							'#FunWooFact: There are more than 150 WooCommerce meetups held all over the world! A great way to meet fellow store owners.'
+						subtitle: (
+							<>
+								<strong>{ __( '#FunWooFact: ' ) }</strong>
+								{ __(
+									'There are more than 150 WooCommerce meetups held all over the world! A great way to meet fellow store owners.'
```
Comment (rjchow) [inline client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/hooks/use-processing-loading-messages.tsx]:
```suggestion
									'There are more than 150 Woo meetups held all over the world! A great way to meet fellow store owners.'
```

**Review by moon0326 — APPROVED** [review]
LGTM 👍 🚀

**Review by chihsuan — APPROVED** [review]
Thanks, looks great! 🚀

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7847598

Hi @rjchow, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `#FunWooFact: `
- `Did you know that Woo powers almost 4 million stores worldwide? You're in good company.`
- `Building the foundations`
- `Did you know that Woo was founded by two South Africans and a Norwegian? Here are three alternative ways to say "store" in those countries - Winkel, ivenkile, and butikk.`
- `Organizing the stock room`
- `Are you Team Cat or Team Dog? The Woo team is split 50/50!`
- `The Woo team is made up of over 350 talented individuals, distributed across 30+ countries.`
- `Opening the doors`
- `Our favorite color is purple `

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7847598).

**Pull request merged**